web/src/lib/components/shared-components/search-bar/search-bar.svelte

```diff
@@ -113,58 +125,20 @@
       </div>
     {/if}
 
-    {#if showBigSearchBar}
-      <div
-        transition:fly={{ y: 25, duration: 250 }}
-        class="absolute w-full rounded-b-3xl border border-gray-200 bg-white pb-5 shadow-2xl transition-all dark:border-gray-800 dark:bg-immich-dark-gray dark:text-gray-300"
-      >
-        <div class="flex px-5 pt-5 text-left text-sm">
-          <p>
-            Smart search is enabled by default, to search for metadata use the syntax <span
-              class="rounded-lg bg-gray-100 p-2 font-mono font-semibold leading-7 text-immich-primary dark:bg-gray-900 dark:text-immich-dark-primary"
-              >m:your-search-term</span
-            >
-          </p>
-        </div>
+    <!-- SEARCH HISTORY BOX -->
+    {#if showHistory}
+      <SearchHistoryBox
+        on:clearAllSearchTerms={clearAllSearchTerms}
```

Comment:
I would probably prefer an interface of:
```ts
clear: string?
```

Omitting the term implies clear all. And then just have `on:clear={({ detail: term }) => handleClear(term)}`

Reply:
One is for clearing the individual search term, one is for clearing all of the terms.

Are you thinking that if we do not pass in the term, we will clear all?

Reply:
Yes